exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>I really like this whole section, it can be overwhelming to examine new data files but this pinpointed me exactly to the things I need to know - what are the columns, what are the units! The one remaining question I have are on some of the choices for this particular cell. Can you add some more detail on why 54-61 were chosen in particular? Are these things the reader will use as they continue through the tutorial? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>